### DISCUSSION TITLE
Slope, aspect, and new function for reprojecting GEBCO

### DISCUSSION THREAD

**Discussion by sean-rohan-NOAA** [noaa-afsc/efhbathy#1] [Announcements]
@MargaretSiple-NOAA @JensMNielsen I added a [new function](https://github.com/afsc-gap-products/akgfmaps/blob/cafed34843294f2c7cc75193eba85c9dc8fbe73e/R/reproject_gebco.R) to the akgfmaps package (version 2.3.0) to generalize the process of reprojecting a GEBCO raster. The function also calculates slope and aspect from the bathymetry.

The [make_gebco_bathymetry.R script](https://github.com/sean-rohan-NOAA/efhbathy/blob/f825de76c942eba90aab50a175e4ccd551322c91/R/make_gebco_bathymetry.R) in this repo has been updated to run the function and write outputs to .grd files ([files here](https://github.com/sean-rohan-NOAA/efhbathy/tree/main/output/EBS)).

**Comment by MargaretSiple-NOAA** [comment]
Sweet! We will use this for the bathymetry raster when we're doing the prey maps.
